src/js/components/Resize.jsx

```diff
@@ -1,12 +1,35 @@
+var _             = require('lodash');
 var React         = require('react/addons');
 var Reflux        = require('reflux');
 var ResizeActions = require('./../actions/ResizeActions');
 
-
 var Resize = React.createClass({
     mixins: [Reflux.ListenerMixin],
 
     render: function () {
+        var select = "";
+
+        if (_.isArray(this.props.presets)) {
```

Comment:
You should use http://facebook.github.io/react/docs/reusable-components.html#prop-validation